src/graph/backend/dnnl/dnnl_partition_impl.cpp

```diff
@@ -113,7 +113,7 @@ status_t dnnl_partition_impl_t::compile(
         compiled_partition_t *compiled_partition,
         const std::vector<logical_tensor_t> &inputs,
         const std::vector<logical_tensor_t> &outputs,
-        const engine_t *g_engine) const {
+        engine_t *g_engine) const {
```

Comment:
What does this mean? We do have a `const` engine in the public compile API. I assume that const-ness will propagate from API to internal implementations.

Reply:
Why do we need to remove `const` here? Also I see both `reset_engine(engine_t *)` and `reset_engine(const engine_t *)` across the PR.